internal/log/log.go

```diff
@@ -121,8 +121,8 @@ func (l *Logger) PopPrefix() {
 	l.prefixStack = l.prefixStack[:len(l.prefixStack)-1]
 }
 
-// quotedCmd returns a concatenated, quoted form of cmd's cmdline
-func quotedCmd(cmd *exec.Cmd) string {
+// QuotedCmd returns a concatenated, quoted form of cmd's cmdline
+func QuotedCmd(cmd *exec.Cmd) string {
```

Comment:
Since you're not using this in `internal/exec/util`, this doesn't need to be changed.

Reply:
I'm calling `log.QuotedCmd(cmd)` in 
```
u.Info("checking if user \"%s\" exists: %s", c.Name, fmt.Errorf("[Attention] %v: Cmd: %s Stdout: %s", err, log.QuotedCmd(cmd), stdout))
```

Reply:
Ah yup, missed that. Nevermind on this.